### DISCUSSION TITLE
Unites / Values

### DISCUSSION THREAD

**Discussion by DrKhurram** [Spine-project/Spine-Toolbox#2180] [Discussion]
hi,
Can i ask in Spine DB. What are the units for everything. Are these SI units? Because i couldn't find any units next to the values.

**Comment by jkiviluo** [comment]
If talking about SpineOpt models, then the units have not been stated explicitly since you could use different unit systems in your data. The important thing is to be consistent in the units you use for a particular model.

That said, typical units would be MW, MWh and currency/MWh. For several parameters the unit is 'p.u.' or per unit which means a value between 0 and 1. This is used so that it can be multiplied by e.g. capacity of a single unit and the number of units, which can change in investment models.


However, if talking about Spine Toolbox in general, then it's up to the user to define what units a particular dataset should use (or if it should be left open as in the case of SpineOpt above).
